pages/planetusermanual.md

```diff
@@ -32,3 +32,72 @@ Under the PLANET system, there are two types of structures: **Nations** and **Co
   * A **Community** is not able to connect to other **Communities**.
 
 The **nations** may store thousands of Resources in many formats and languages.  Unique collections of Resources can be selected and downloaded into each **Community** from the **Nation**.  This gives a Learner a personalized approach to his or her education.
+
+## How do I navigate and use PLANET?
+###_How do I access PLANET and become a member?_
```

Comment:
@waryhermit  It looks like this only on this preview, In actual when I checked using this link, looked fine. Anyways, should I change it? https://rawgit.com/Supriyaranjan/Supriyaranjan.github.io/Supriyaranjan-patch-8/#!pages/planetusermanual.md
![image](https://cloud.githubusercontent.com/assets/26332034/24641079/02afdce4-18c2-11e7-9d67-8367cac6e452.png)